fusor-ember-cli/app/routes/deployments.js

```diff
@@ -1,8 +1,26 @@
 import Ember from 'ember';
+import PaginationRouteMixin from "../mixins/pagination-route-mixin";
+import _ from 'lodash/lodash';
 
-export default Ember.Route.extend({
-  model() {
-    return this.store.findAll('deployment');
+export default Ember.Route.extend(PaginationRouteMixin, {
+
+  model(params) {
+    // server-side deployments controller uses scoped search params[:order] for sorting
+    let sort_by = params['sort_by'] || 'updated_at';
+    let dir = params['dir'] || 'DESC';
+    let page = params['page'] || 1;
+    params['order'] = sort_by + ' ' + dir;
+    let controller = this.controllerFor('deployments');
+    return this.store.query('deployment', params);
+  },
+
+  setupController(controller, model) {
+    controller.set('model', model);
+    controller.set('totalCnt', model.get('meta.total'));
+    controller.set('pageNumber', model.get('meta.page'));
+    controller.set('totalPages', model.get('meta.total_pages'));
+    controller.set('pageRange', _.range(1, model.get('meta.total_pages') + 1));
```

Comment:
I'm hesitant about adding the entire lodash library for this call since there's so much overlap with Ember's enumerable.  We could create a CollectionUtils with a range function and avoid it if you guys want.  Not a big deal, but just thought I'd mention it.